code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -309,6 +314,44 @@
 	M.adjustBrainLoss(5, 150)
 	holder.remove_reagent(id, 1)
 
+/datum/reagent/fuel/holyoil		//Its oil
+	name = "Zelus Oil"
+	id = "holyoil"
+	description = "Oil blessed by a grater being."
+	taste_description = "oils"
```

Comment:
"oils" -> "oil" (doesn't need to be a plural)

Reply:
oils as in different types, this one at lest was intented